docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,210 @@
+- Optionally, use the **Pull Secret File** field to upload the pull secret you received from Yugabyte Support. 
+
+![OpenShift Provider Config](/images/ee/openshift-cloud-provider-setup.png)
+
+- Click **Add Region** and complete the **Add new region** dialog shown in the following illustration by first selecting the region you found previously (US East), and then entering the following information:
+  - In the **Zone** field, enter the exact zone label (us-east4-a).
+  - In the **Namespace** field, enter yb-platform.
+
+![img](https://lh5.googleusercontent.com/gQ-fTRZnDTp4kCqNaKo8KUgVA2mJeCCjaHiqzdCOcG4350yxgDGZojMWhsfdvcpLJbOio8sL8K932wmDM6_S8fIkL9wfKAo-b3340Yvc1Dy4FJ61o2Ec93DptkB_Ski1XqaU3UrT)
+
+- Click **Add Region**.
+- Click **Save**. 
+
+You should see the newly-added provider under **Red Hat OpenShift configs**.
+
+## Creating a Universe Using the Provider
+
+You can create a universe using the provider as follows:
+
+- Use the Yugabyte Platform web UI to navigate to **Universes** and then click **Create Universe**. 
+
+- Complete the **Create Universe** page shown in the following illustration by entering the following information:
+
+  - In the **Name** field, enter universe-1.
+  - In the **Provider** field, enter ocp-test.
+  - In the **Regions** field, enter US East.
+  - In the **Instance Type** field, enter xsmall (2 cores, 4GB RAM).
+
+![img](https://lh3.googleusercontent.com/E8EwHd0olx_ZmmpRN-vpem8TAvqqU_gPQGyuMJH9yDxKS89cswTURFASkAE0fDKWMCfImUknxbyN7z87SzaOJJE_PO6Wo4aKw7UHIdXy7oCxPgTmv2XvRMvAIvbRQygS0gz3oXNz)
+
+- Click **Create**. 
+
+The following illustration shows the universe creation progress:
+
+![img](https://lh3.googleusercontent.com/wmVq3lUnqseriSgXcw-gwqNRct4CxUSZ0NnGwgVJ5UG-7GU8Ja4kKHGUZ428BmYOZvcemMLlod_9PBWQAFaj99bOybyW_XEc40X6mZL9fZKi6IQ9q7-Cr52XTfzjmw0ppGTN8b9Z)
+
+If you click universe-1 and then navigate to **Tasks**, you should see a page similar to the one shown in the following illustration:
+
+![img](https://lh3.googleusercontent.com/UFeWUHRm7Amxek1U8yxqeV3z4RiUll05Vo9VowiCImlELaCabHyaDFFT81XYbbBXUMoDAK8g13q9FP4ZbwOSAoCzsqGV83zqF9Jkcw_3pb4U4WxgC7MOiULbCpAArxIqSHMrDDIC)
+
+Upon successful creation of the universe, the **Overview** tab of universe-1 should look similar to the following:
+
+![img](https://lh5.googleusercontent.com/TpdExY6idrjH64UXfXKsY9RRwPaLU5OWfNHD_VZxrzRDWAXmG_dftROfNyxRX7TNWr7BgWxrsuZXOa4N_4KP9RHt4c9AgleCGSmziKudtIbqzrJAqOu7YL0oa9MpfdCmBvNE7rPL)
+
+## Troubleshooting the Universe Creation
+
+If the universe creation remains in Pending state for more than 2-3 minutes, open the OCP web console, navigate to **Workloads > Pods** and check if any of the pods are in pending state, as shown in the following illustration:
+
+![img](https://lh5.googleusercontent.com/jHJnEpQ6baGDDuppWND7O6z2SZM4dOPkIFekGa556xUffhOjEFdkM3EUvL2BEQXponEkEjYOwIFmPKu3z35-d903Vhz0D2i_6MyZEXstkrHN1jpgQQFgFy675cNwyzhonpGf65l9)
```

Comment:
ocp-yugabyte-platform-tshoot-1.png

Reply:
I did replace all the images (they no longer point to google). This change will be visible when I submit the PR.